src/pegin/components/create/PegInForm.vue

```diff
@@ -144,16 +144,11 @@ export default defineComponent({
     const selectedAccountBalance = useGetter<SatoshiBig>('pegInTx', constants.PEGIN_TX_GET_SELECTED_BALANCE);
 
     const enoughAmountFlyover = computed(() => {
-      const quote = selectedQuote.value?.quote;
-      if (!quote) {
+      if (!selectedQuote.value) {
         return false;
       }
-      const fullAmount: SatoshiBig = quote?.value
-        .plus(quote.productFeeAmount)
-        .plus(quote.callFee)
-        .plus(SatoshiBig.fromWeiBig(quote.gasFee))
+      const fullAmount: SatoshiBig = selectedQuote.value.totalValueToTransfer
```

Comment:
insert selectedFee on `selectedQuote` object, and then call getFullAmount from `selectedQuote` object